program/src/state.rs

```diff
@@ -83,12 +83,13 @@ pub enum DataType {
     ReferrerIdRecord,
 }
 
-const NUM_HEALTHS: usize = 2;
+const NUM_HEALTHS: usize = 3;
 #[repr(usize)]
 #[derive(Copy, Clone, IntoPrimitive, TryFromPrimitive)]
 pub enum HealthType {
     Maint,
     Init,
+    Equity,
```

Comment:
Could we please add one line descriptions to all these variants?

Reply:
Still not sure, what equity means in our context 😓

Reply:
Looks like it's the USD value of assets - liabilities. (all weights are 1)